sdk/core/Azure.Core/src/Pipeline/BearerTokenAuthenticationPolicy.cs

```diff
@@ -28,16 +30,21 @@ public class BearerTokenAuthenticationPolicy : HttpPipelinePolicy
         /// Creates a new instance of <see cref="BearerTokenAuthenticationPolicy"/> using provided token credential and scopes to authenticate for.
         /// </summary>
         /// <param name="credential">The token credential to use for authentication.</param>
-        /// <param name="scopes">Scopes to authenticate for.</param>
+        /// <param name="scopes">_scopes to authenticate for.</param>
         public BearerTokenAuthenticationPolicy(TokenCredential credential, IEnumerable<string> scopes)
-            : this(credential, scopes, TimeSpan.FromMinutes(5), TimeSpan.FromSeconds(30)) { }
+            : this(credential, scopes, TimeSpan.FromMinutes(5), TimeSpan.FromSeconds(30))
+        { }
 
-        internal BearerTokenAuthenticationPolicy(TokenCredential credential, IEnumerable<string> scopes, TimeSpan tokenRefreshOffset, TimeSpan tokenRefreshRetryDelay)
+        internal BearerTokenAuthenticationPolicy(TokenCredential credential,
```

Comment:
Nit: IMO, if you're going to put params on separate lines, it's typical to start with the first param - makes it easier to notice them all.

Reply:
I've been playing with editor settings and I need to tweak this - thanks